cpu/esp_common/Makefile.include

```diff
@@ -105,3 +105,45 @@ PROGRAMMER ?= esptool
 
 # Add esptool in the list supported programmers
 PROGRAMMERS_SUPPORTED += esptool
+
```

Comment:
Shouldn't openocd be added above?

Reply:
This file is shared between esp8266 and esp32. Only esp32 cpus with the board having jtag are supported - so it is added inside the if checking esp_jtag module there (cpu/esp32/Makefile.include)

Reply:
Ok, then follow up the question what is the idea of the `esp_jtag` module? Why do you need it, couldn't you simply do `PROGRAMMER=openocd`, and couldn't the infeq be against PROGRAMMER so: `ifneq(,$(filter openocd,$(PROGRAMMER))`?.

Reply:
On the esp32-ethernet-kit board there are not enough pins, so enabling JTAG blocks SPI:

https://github.com/RIOT-OS/RIOT/blob/master/boards/esp32-ethernet-kit-v1_0/include/periph_conf.h#L105